### ISSUE TITLE
Support other Postgres relevant Sequelize options

### ISSUE THREAD

**Issue by davesag** [davesag/sequelize-pg-utilities#32]
Sequelize supports [a wide range of options](https://github.com/sequelize/sequelize/blob/master/lib/sequelize.js#L55) that this utility ignores.  Go through them, work out which ones are supported by `Postgres` and allow them to be used.

## programatic only options

Can only be set by passing in as an option to Sequelize via

```js
const sequelize = new Sequelize('database', 'username', 'password', options);
```

- [x] `dialectModule`
- [x] `dialectModulePath`
- [x] `define`
- [x] `query`
- [x] `schema`
- [x] `set`
- [x] `sync`
- [x] `pool.validate`
- [x] `quoteIdentifiers`
- [x] `transactionType`
- [x] `retry.match`
- [x] `retry.max`
- [x] `typeValidation`
- [x] `hooks`

## config options

Can be defined in config file.

- [x] `protocol`
- [x] `timezone`
- [x] `clientMinMessages`
- [x] `benchmark`
- [x] `omitNull`
- [x] `native`
- [x] `replication`
- [x] `pool.acquire`
- [x] `pool.evict`

## environment variable options

Can be defined as an environment variable

- [x] `DB_POOL_ACQUIRE` ie `pool.acquire`
- [x] `DB_POOL_EVICT` ie `pool.evict`

omiting any of these options means they are omitted from the final config object.

**Comment by davesag** [comment]
## programatic only options

* `dialectModule`
* `dialectModulePath`
* `define`
* `query`
* `schema`
* `set`
* `sync`
* `pool.validate`
* `quoteIdentifiers`
* `transactionType`
* `retry.match`
* `retry.max`
* `typeValidation`
* `hooks`

## config options

* `protocol`
* `timezone`
* `clientMinMessages`
* `benchmark`
* `omitNull`
* `native`
* `replication`
* `pool.acquire`
* `pool.evict`

## environment variable options

* `DB_POOL_ACQUIRE` ie `pool.acquire`
* `DB_POOL_EVICT` ie `pool.evict`

omiting any of these options means they are omitted from the final config object.

**Comment by adlenafane** [comment]
We'd love to have the `pool.acquire` option @hiboo.io. Could I help you on this topic?

**Comment by davesag** [comment]
Hey @adlenafane 

> We'd love to have the `pool.acquire` option @hiboo.io. Could I help you on this topic?

I have added this in PR #67 

Please have a look and let me know if this works for you.

**Comment by davesag** [comment]
@adlenafane support for `pool.acquire` and `pool.evict` have been added to Version 1.2.4 - released to NPM a few minutes ago.  Thanks for your input.

**Comment by davesag** [comment]
Implemented in version 1.3.0
